From: Paul Novotny <novotny@righthandrobotics.com>
Date: Mon, 20 Mar 2017 14:35:58 -0400
Subject: Reduce the minimum required cmake version

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 159fcce..d183ff3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 2.8.12)
 
 project(CatchSelfTest)
 
